packages/node/test/index.test.ts

```diff
@@ -1,13 +1,17 @@
+import type { OutgoingLogBody } from '../src/lib/metrics-log';
 import type { Express } from 'express';
+import type { Headers } from 'headers-polyfill';
 
 import * as crypto from 'crypto';
 import { createServer } from 'http';
 
 import chai, { expect } from 'chai';
+import chaiAssertionsCount from 'chai-assertions-count';
```

Comment:
had to bring this package in since mocha/chai doesn't have an equivalent for `expect.assertions` 🙃